code/src/components/Styles.scss

```diff
@@ -0,0 +1,185 @@
+/*** VARIABLES ***/
+
+$gray: #f2f0f0;
+$black: #000;
+$pink: #f4acac;
+$light-pink: #ffd5cd;
+$red: red;
+$green: green;
+
+/*** GENERAL ***/
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  font-family: menlo;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*** APP ***/
+
+.App {
+  margin: 50px auto;
+  width: 90%;
+  max-width: 500px;
+
+  &__grid {
+    display: grid;
+    row-gap: 50px;
+  }
+}
+
+/*--- FORM ---*/
+
+.Form {
+  display: grid;
+  grid-template-rows: min-content;
+  grid-row-gap: 5px;
+  align-items: center;
+  padding: 20px;
+  border: 1px solid $black;
+  box-shadow: 10px 10px $black;
+  background: $gray;
+
+  &__header {
+    font-size: 18px;
+    font-weight: 400;
+    margin-top: 0;
+  }
+
+  &__textarea {
+    width: 100%;
+    padding: 5px;
+
+    &::-webkit-input-placeholder {
+      color: $pink;
+    }
+
+    &:-ms-input-placeholder {
+      color: $pink;
+    }
+
+    &::placeholder {
+      color: $pink;
+    }
+  }
+
+  &__text {
+    justify-self: flex-end;
+    font-size: 12px;
+    padding: 0;
+    margin-top: 0;
+
+    &--green {
+      color: $green;
+    }
+
+    &--red {
+      color: $red;
+    }
+  }
+}
+
+/*--- ARTICLE ---*/
+
+.Article {
+  display: grid;
+  grid-template-rows: min-content;
+  grid-template-columns: min-content 1fr 1fr;
+  grid-row-gap: 5px;
+  grid-column-gap: 10px;
+  align-items: center;
+  padding: 20px;
+  border: 1px solid $black;
+  box-shadow: 10px 10px $black;
+  word-wrap: break-word;
+
+  &__text {
+    grid-column: 1/4;
+  }
+
+  &__text--likes {
+    font-size: 13px;
+    opacity: 0.6;
+    margin: 5px 0;
+  }
+
+  &__text--time {
+    font-size: 13px;
+    opacity: 0.4;
+    text-align: right;
+  }
+
+  &__span {
+    animation: pulse 3s ease-in-out infinite;
+  }
```

Comment:
this is so subtle that I cannot really see it. Consider making it pop more for effect 😄